src/integration-tests/src/storage.rs

```diff
@@ -102,3 +146,14 @@ async fn cleanup_bucket(client: storage::client::Storage, name: String) -> Resul
     let _ = futures::future::join_all(pending).await;
     client.delete_bucket(&name).send().await
 }
+
+pub(crate) fn random_bucket_id() -> String {
+    const CHARSET: &[u8] = b"abcdefghijklmnopqrstuvwxyz0123456789";
+    const PREFIX: &str = "rust-sdk-testing-";
+    let mut bucket_id = String::new();
+    for _ in 0..(BUCKET_ID_LENGTH - PREFIX.len()) {
+        let idx = rand::rng().random_range(0..CHARSET.len());
+        bucket_id.push(CHARSET[idx] as char);
+    }
```

Comment:
Consider:

```suggestion
    let id = std::iter::from_fn(|| move { rand::rng().random_range(0..CHARSET.len()) })
        .take(BUCKET_ID_LENGTH - PREFIX.len())
        .map(|idx| char::from(CHARSET[idx])
        .collect();
```

Or if you want to be nice, add an analog to `Alphanumeric` for lowercase characters to `lib.rs`.

Reply:
You are now the proud owner of `RandomChars` which impls `rand::distr::Distribution<u8>` for any set of characters we may care about.